bundle/tests/scorecard/kuttl/sidecar-deployment/02-assert.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: Pod
+spec:
+  containers:
+  - name: app
```

Comment:
It'll be good to add the label `app.kubernetes.io/instance` to ensure that it's the right Pod and not a Pod leftbehind from previous test scenarios (i.e. init-container)